app/controllers/api/v4/converts_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
     end
 
     old_extension = Run.program(@run.program).file_extension
-    new_extension = Run.program(params[:format]).file_extension
+    new_extension = Run.program(params[:format]).try(:file_extension) || params[:format]
```

Comment:
when does `Run.program` return nil? afaict `check_parameters` takes care of unexpected values